website/content/en/guides/level-up/e2e-ack-guide.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Ensure data is always successfully delivered
+short: End-to-end acknowledgement
+description: Learn how to use end-to-end acknowledgement
+author_github: https://github.com/barieom
+domain: delivery
+weight: 5
+tags: ["delivery", "logs", "level up", "guides", "guide"]
+---
+
+{{< requirement >}}
+Before you begin, this guide assumes the following:
+
+* You understand the [basic Vector concepts][concepts]
+* You understand [how to set up a basic pipeline][pipeline]
+
+[concepts]: /docs/about/concepts
+[pipeline]: /docs/setup/quickstart
+{{< /requirement >}}
+
+
+Vector is often deployed in mission critical environments where loss of data is
+non-negotiable. Ergo, assurance that every data flowing through your Vector
+pipeline is successfully delivered to the end destination is critical.
+
+We're excited to walk you through exactly you can accomplish this by using
+Vector's end-to-end acknowledgement feature.
+
+## Getting Started
+
+You can set and control the acknowledgement feature either at the global level or
+the source level. Let's start with the global configuration option first, as it is
```

Comment:
I thought we swapped this to be enabled at the `sink` side

Reply:
Yeah, that is correct, acknowledgements are configured on sinks now instead of sources. When acknowledgements are enabled for a sink, all associated sources that support acknowledgements will wait for delivery. The ones that don't support it, will output a warning message (we should call that out so users know what to look for).